gamestonk_terminal/stocks/stocks_controller.py

```diff
@@ -233,6 +235,25 @@ def call_news(self, other_args: List[str]):
             default=5,
             help="Number of latest news being printed.",
         )
+        parser.add_argument(
+            "-d",
+            "--date",
+            action="store",
+            dest="n_start_date",
+            type=valid_date,
+            default=datetime.now() - timedelta(days=7),
+            help="The starting date (format YYYY-MM-DD) to search articles from",
+        )
+        parser.add_argument(
+            "-s",
+            "--sort",
+            action="store",
+            choices=["newest", "oldest"],
+            dest="n_sort",
+            default="newest",
+            help="Sort by newest articles first or oldest first (default newest)",
+        )
```

Comment:
You should have this be a true/false to indicate reverse instead of using ns_parser.n_sort=="newest".  So this would be default=True and action = "store_false"

Reply:
Got you